LayoutTests/platform/gtk/accessibility/roles-exposed-expected.txt

```diff
@@ -223,6 +223,12 @@ mark
 math
       AXRole: AXMath
 
+menu
+      AXRole: AXList
+
+li
+      AXRole: AXListItem
+
```

Comment:
So I believe it should be placed after all the MathML elements here. I don't see the meter element in the output so maybe it's not up-to-date.